File: restartonempty/src/main/java/uk/asheiou/restartonempty/ROECommandExecutor.java
package uk.asheiou.restartonempty;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import net.md_5.bungee.api.ChatColor;

public class ROECommandExecutor implements CommandExecutor {
  @Override
  public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
    if (args != null && args.length > 0) {
      if (args.length > 1) {
        sender.sendMessage(ChatColor.RED + "Too many arguments! Usage:");
        return false;
      }
      switch (args[0].toLowerCase()) {
      case "true":
        ROEToggle.setStatus(true);
        sender.sendMessage(ChatColor.GREEN + "The server will restart on empty.");
        return true;
      case "false":
        ROEToggle.setStatus(false);
        sender.sendMessage(ChatColor.GREEN + "The server will no longer restart on empty.");
        return true;
      case "status":
        sender.sendMessage(ChatColor.GREEN + "The server " + (ROEToggle.getStatus() ? "will" : "will not") + " restart on empty.");
        return true;
      default:
        sender.sendMessage(ChatColor.RED + "Argument not recognised! Usage:");
        return false;
      }
    }
    boolean toSet = !ROEToggle.getStatus();
    ROEToggle.setStatus(toSet);
    sender.sendMessage(ChatColor.GREEN + "The server " + (toSet ? "will" : "will no longer") + " restart on empty.");
    return true;
  }
}

File: src/main/kotlin/cymru/asheiou/restartonempty/ROECommandExecutor.kt
package cymru.asheiou.restartonempty

import net.md_5.bungee.api.ChatColor
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import java.util.Locale.getDefault

class ROECommandExecutor : CommandExecutor {
  override fun onCommand(
    sender: CommandSender,
    command: Command,
    label: String,
    args: Array<out String?>) : Boolean{
    if (args.isNotEmpty()) {
      if (args.size > 1) {
        sender.sendMessage("${ChatColor.RED}Too many arguments! Usage:")
        return false
      }
      when (args[0]?.lowercase(getDefault())) {
        "true" -> {
          ROEToggle.status = true
          sender.sendMessage("${ChatColor.GREEN}The server will restart on empty.")
          return true
        }

        "false" -> {
          ROEToggle.status = false
          sender.sendMessage("${ChatColor.GREEN}The server will no longer restart on empty.")
          return true
        }

        "status" -> {
          sender.sendMessage("${ChatColor.GREEN}The server " + (if (ROEToggle.status) "will" else "will not") + " restart on empty.")
          return true
        }

        else -> {
          sender.sendMessage("${ChatColor.RED}Argument not recognised! Usage:")
          return false
        }
      }
    }
    ROEToggle.status = !ROEToggle.status
    sender.sendMessage("${ChatColor.AQUA}The server " + (if (ROEToggle.status) "will" else "will no longer")
            + " restart on empty.")
    return true
  }
}